ui/components/app/asset-list-item/asset-list-item.js

```diff
@@ -98,11 +98,10 @@ const AssetListItem = ({
         <button
           className="asset-list-item__token-button"
           onClick={onClick}
-          title={`${primary} ${tokenSymbol}`}
+          title={primary}
         >
           <h2>
             <span className="asset-list-item__token-value">{primary}</span>
-            <span className="asset-list-item__token-symbol">{tokenSymbol}</span>
```

Comment:
In some other places, we intentionally put the symbol and the value in separate elements so that we can use CSS to ellipse long values without obscuring the symbol. That seems like it would be useful here too :thinking: But that said I can't see any evidence of such a strategy being used here.

Reply:
Added back!